apps/koala-frontend/src/app/core/components/footer/footer.component.html

```diff
@@ -1,7 +1,7 @@
 <footer class="footer">
   <p-divider></p-divider>
   <div class="footer-container">
-    <p-button label="Copyright KoALa 2023" styleClass="p-button-link"></p-button>
+    <p-button label="© Koala App 2023/2024" styleClass="p-button-link" [routerLink]="['github']"></p-button>
```

Comment:
was there a reason to introduce a separate "github" route which also navigates to the about page?

Reply:
Actually not really. Originally, I wanted to navigate to github directly. And later I thought it might be a nice to have for some additional navigation On the other hand this copyright text could be just text, but this was too hard to layout.

Reply:
OK, so then just reuse the "about page" route for now as long as the navigation goes to this page and we'll adapt later when there is a different target